tests/test_kube_state_deployment.py

```diff
@@ -54,3 +54,21 @@ def test_kube_state_deployment_custom_resources(self, kube_version):
             "limits": {"cpu": "777m", "memory": "999Mi"},
             "requests": {"cpu": "666m", "memory": "888Mi"},
         }
+
+    def test_kube_state_deployment_with_default_args(self, kube_version):
```

Comment:
We should have another test here `test_kube_state_deployment_with_custom_extraArgs` that overrides the defaults and makes sure the default values do not show up.

Reply:
I guess this isn't the right ticket for that.